CHANGELOG.md

```diff
@@ -18,11 +18,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### `Dependencies`
```

Comment:
Missing descriptions of new flags and also default parameter changes (mmseqs searchrtpye)